hack/test-end-to-end-docker.sh

```diff
@@ -65,8 +65,7 @@ function cleanup()
 	exit $out
 }
 
-trap "exit" INT TERM
-trap "cleanup" EXIT
+trap "cleanup" EXIT INT TERM
```

Comment:
Compare to this one: https://ci.openshift.redhat.com/jenkins/job/merge_pull_requests_origin/4697/consoleText